webpack.common.config.js

```diff
@@ -1,72 +1,100 @@
-var webpack = require('webpack');
-var path    = require('path');
-var empty = 'widget/empty';
-var packageJson = require('./package.json');
+var path      = require('path');
+var EMPTY     = 'widget/empty';
+var TARGET_JS = path.resolve(__dirname, 'target/js/');
 
-module.exports = {
-  entry: './target/js/widget/OktaSignIn.js',
-  output: {
-    path: path.resolve(__dirname, 'target/js/'),
-    filename: 'okta-sign-in.js',
-    library: 'OktaSignIn',
-    libraryTarget: 'umd'
-  },
-  resolve: {
-    root: [
-      path.resolve(__dirname, 'target/js')
-    ],
-    alias: {
-      'handlebars': 'handlebars/dist/handlebars',
-      'duo': 'vendor/Duo-Web-v2',
-      'xdomain': 'vendor/xdomain-0.7.5',
-      'nls': '@okta/i18n/dist/json',
-      'okta': 'shared/util/Okta',
-      'shared/util/Bundles': 'util/Bundles',
+// Return a function so that all consumers get a new copy of the config
+module.exports = function (outputFilename) {
+  return {
+    entry: './target/js/widget/OktaSignIn.js',
+    output: {
+      path: TARGET_JS,
+      filename: outputFilename,
+      library: 'OktaSignIn',
+      libraryTarget: 'umd'
+    },
+    resolve: {
+      root: [TARGET_JS],
+      alias: {
+        // General remapping
+        'nls': '@okta/i18n/dist/json',
```

Comment:
That's one of the nice things about webpack (and the reason to switch from requireJs) - it knows about the `node_modules` folder and how to resolve module paths. So in this case, it resolves it against `resolve.root` + `node_modules`